src/components/PeopleManagement/DownloadCSVButton.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import {
+  Toast, StatefulButton, Icon, Spinner, useToggle,
+} from '@openedx/paragon';
+import { Download, Check } from '@openedx/paragon/icons';
+import { logError } from '@edx/frontend-platform/logging';
+import { downloadCsv } from '../../utils';
+
+const csvHeaders = ['Name', 'Email', 'Joined Organization', 'Enrollments'];
+
+const dataEntryToRow = (entry) => {
+  const { enterpriseCustomerUser: { name, email, joinedOrg }, enrollments } = entry;
+  return [name, email, joinedOrg, enrollments];
+};
+
+const getCsvFileName = () => {
+  const padTwoZeros = (num) => num.toString().padStart(2, '0');
+  const currentDate = new Date();
+  const year = currentDate.getUTCFullYear();
+  const month = padTwoZeros(currentDate.getUTCMonth() + 1);
+  const day = padTwoZeros(currentDate.getUTCDate());
+  return `${year}-${month}-${day}-people-report.csv`;
+};
+
+const DownloadCsvButton = ({ testId, fetchData, totalCt }) => {
+  const [buttonState, setButtonState] = useState('pageLoading');
+  const [isOpen, open, close] = useToggle(false);
```

Comment:
Change this to something more informative like [isToastOpen, openToast, and closeToast]

Reply:
Good call!